### DISCUSSION TITLE
Feature request: Share image to existing issue

### DISCUSSION THREAD

**Discussion by philwingfield** [community/community#27] [Mobile]
Hi there, 

I can already use the built in "share" feature of android to send screenshots from my phone to a new issue on a repository. I would like the ability to post them on a comment to an existing issue as well. This would be hugely useful for anyone working on a mobile app I would have thought. Often there is already a relevant issue, and I want to add supporting imagery to it, rather than create a new issue. Currently I have to save the image and upload it manually, which is a bit clunkier.

Given the existing functionality I think this ought to be easy enough to implement.

The existing workflow that you currently support is this:

1. Open an image/take a screenshot/download an image etc.
2. "Share" -> Share to apps -> More
3. Select "Github: create issue"
4. Select a repository on which to create the issue
5. A blank issue is created, with the image uploaded and inserted as markup to the issue description.

The workflow I would like is very similar:

1. Open an image/take a screenshot/download an image etc.
2. "Share" -> Share to apps -> More
3. Select "Github: attach to issue"
4. Select a repository
5. Select an issue
6. A comment is added to the issue, with the image uploaded and inserted as markup in the comment.

Thanks very much,

Phil

**Comment by candyho** [comment]
Hey @philwingfield , thanks for your suggestion! Agree that this would make it easier to document things for an existing topic on the go. I had a quick chat with our engineer and the flow you proposed is definitely possible. I'll add this idea to our improvement backlog.
